api/src/services/nfl-players.service.ts

```diff
@@ -18,12 +18,32 @@ export class NFLPlayerService {
     return findNFLPlayer;
   }
 
+  public async findNFLPlayersByFilter(positions: string[]): Promise<NFLPlayer[]> {
+    const findNFLPlayers: {
+      rows: NFLPlayer[],
+      count: number
+    } = await DB.NFLPlayers.findAndCountAll({
+      where: {
+        position: {
+          [DB.Sequelize.Op.in]: positions
```

Comment:
Is there a reason to use operators here? I was reading some of the sequelize docs and they mentioned that sequelize can infer the use of operators? I'm not entirely sure